### PR TITLE
FFI: Add function rnp_key_is_expired().

### LINKED ISSUE #1532: FFI: add function rnp_key_is_expired().
## Description

While currently user may manually check key's creation and expiration time, it would be handy to have a shortcut function instead of 3 other calls + calculations.

### PR DESCRIPTION
**Pull request by ni4**
Fixes #1532

### CODE REVIEW

File: src/lib/rnp.cpp [hunk 1]
```diff
@@ -6899,6 +6899,21 @@ try {
...
 
+rnp_result_t
+rnp_key_is_expired(rnp_key_handle_t handle, bool *result)
+try {
+    if (!handle || !result) {
+        return RNP_ERROR_NULL_POINTER;
+    }
+    pgp_key_t *key = get_key_prefer_public(handle);
+    if (!key) {
+        return RNP_ERROR_BAD_PARAMETERS;
```
Comment (ronaldtse) [inline src/lib/rnp.cpp:6910]: Codecov says this line is not covered by tests.

Comment (ni4) [reply src/lib/rnp.cpp:6910]: Hm, wondering how to make it covered... Will think about.

**Review by antonsviridenko — APPROVED** [review]

**Review by ronaldtse — APPROVED** [review]
@ni4 will let you consider the right time to merge, thanks!

**Comment by ni4** [comment]
Okay, now Codecov is happy so merging this. Thanks!

**Pull request merged**